### DISCUSSION TITLE
How to find out the excluded filters?

### DISCUSSION THREAD

**Discussion by GH01** [uBlockOrigin/uBlock-issues#1982] [Q&A]
In the `Troubleshooting Information`, e.g.:
```
    user-filters: 100-5
    easylist: 6000-50
```

So there are some filters that are excluded.
How can I find out them, especially in the `user-filters`?

Thank you.

**Comment by ghost** [comment]
Some invalid filters will be logged in under "All" in the logger.

**Comment by u-RraaLL** [comment]
The "excluded" filters are duplicates of filters from other lists. Invalid filters shouldn't be counted at all.
